app/assets/stylesheets/provider/_header.scss

```diff
@@ -1,16 +1,15 @@
 .Header {
   background-color: $header-background-color;
-  border-top: line-height-times(1 / 10) solid $header-border-color;
-  height: line-height-times(5 / 2);
-  padding: 0 0 0 0;
+  height: line-height-times(3.15);
```

Comment:
I think we prefer using rational numbers instead of floats

Reply:
Why?